importer/src/path_canonicalizer.rs

```diff
@@ -205,6 +245,8 @@ fn check_path_internal<'a, T>(
                     SymbolValueRef::Import(p) => {
                         location.pop().unwrap();
 
+                        // TODO we do a recursive call here, but shouldn't this actually
```

Comment:
To be handled at a later point. This concerns existing code.

Reply:
Please open an issue

Reply:
Actually I think this is indeed fine.